media/public/include/IMediaPipeline.h

```diff
@@ -38,6 +38,11 @@
 #include "IMediaPipelineClient.h"
 #include <MediaCommon.h>
 
+namespace firebolt::rialto::client
```

Comment:
fwd declarations

Reply:
Added comments in the code because I think these forward declarations might also be necessary